dss/api/bundles/__init__.py

```diff
@@ -132,6 +136,50 @@ def get(
         response.headers['Link'] = link
         return response
 
+@dss_handler
+def enumerate(replica: str, prefix: str, token: str = None,
+              per_page: int = PerPageBounds.per_page_max,
+              search_after: typing.Optional[str] = None):
+
+    storage_handler = Config.get_blobstore_handle(Replica[replica])
+    key_prefix = prefix if prefix else 'bundles/'  # might need to add filtering over here (no files/blobs allowed)
```

Comment:
This should only list bundles. The purpose of the prefix is to allow parallelized bundle listings.
```
key_prefix = f"bundles/{prefix}"
```

Reply:
I just realized that I made a similar comment.  @amarjandu Please address comments like these before asking for additional reviews.